[Bug] Library not usable without skipLibCheck: true?

**Describe the bug**
It seems that I cannot build a TypeScript project that references terra-draw without skipLibCheck enabled in my project. Without that, I get a bunch of TypeScript compilation errors. Here's a sampling:

```
node_modules/terra-draw/dist/extend.d.ts:1:57 - error TS2307: Cannot find module './adapters/common/base.adapter' or its corresponding type declarations.

1 import { TerraDrawBaseAdapter, BaseAdapterConfig } from "./adapters/common/base.adapter";
```

```
node_modules/terra-draw/dist/adapters/google-maps.adapter.d.ts:6:14 - error TS2503: Cannot find namespace 'google'.

6         map: google.maps.Map;
```

```
node_modules/terra-draw/dist/adapters/arcgis-maps-sdk.adapter.d.ts:7:27 - error TS2307: Cannot find module '@arcgis/core/layers/GraphicsLayer' or its corresponding type declarations.

7 import GraphicsLayer from "@arcgis/core/layers/GraphicsLayer";
```

The first error seems to be from referencing a file that isn't provided in the npm bundle (I do not see a common folder, or base.adapter.d.ts anywhere). I'm not sure why that file isn't included?

The other errors appear to come from not having various things installed to support the various adapters. However, I definitely do not want to install OpenLayers and others when I only care about MapLibre. One way something like this is generally solved is to move each adapter into its own npm package to separate out the dependencies.

**Terra Draw npm version**
1.0.0-beta.6

**To Reproduce**
Steps to reproduce the behavior:
1. Create TypeScript project without skipLibCheck: true
2. Install and import terra-draw
3. Build

**Expected behavior**
No build errors!

**Additional context**
Requiring skipLibCheck: true is onerous, and enabling it would not be doable in my project, where I have numerous internal dependencies that I would definitely like to get type-checked.

--- a/.npmignore
+++ b/.npmignore
@@ -9,7 +9,6 @@ data/
 .husky/
 .github/
 ROADMAP.md
-common/
 guides/
 readme.gif
 jest.config.ts

